src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -148,32 +148,43 @@ const WorkspaceAccessError = () => {
   ])
 }
 
-const useClusterPolling = namespace => {
+const useCloudEnvironmentPolling = namespace => {
```

Comment:
Did we decide to do an overall re-name of cluster to cloud environment? Seems to me like consistency would be best so either should change all or maybe keep this as cluster

Reply:
In general we're leaving variables and methods named 'cluster', but since this now deals with both runtimes and disks, proactively changing the name here seems like a good idea.